packages/cra-template/template/.editorconfig

```diff
@@ -0,0 +1,15 @@
+# https://github.com/facebook/react/blob/main/.editorconfig
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+max_line_length = 80
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = 0
```

Comment:
Don't you think we don't need `max_line_length`
Is there a reason why you set zero?

Reply:
It's the config of the main React project: https://github.com/facebook/react/blob/main/.editorconfig

I believe 0 disables max_line_length, so *.md files aren't affected by the 80 characters limit of other files.

Reply:
I wonder why you set it to zero.

Is it the same as when the motion is not set even if it is set to zero?

@ceithir, This is just question!